python/ray/_raylet.pyx

```diff
@@ -748,6 +749,11 @@ cdef void delete_spilled_objects_handler(
                     ray_constants.WORKER_PROCESS_TYPE_RESTORE_WORKER_IDLE)
                 proctitle = (
                     ray_constants.WORKER_PROCESS_TYPE_RESTORE_WORKER_DELETE)
+            elif <int> worker_type == <int> WORKER_TYPE_UTIL_WORKER:
```

Comment:
Why do we need this for delete_spilled_objects?

Reply:
Sorry, I guess I only searched the SPILL_WORKER and try to add it there, but not check them carefully. Thanks!